<launch>

  <arg name="camera_image_topic"                          default="/camera/rgb/image_raw"/>>
  <arg name="camera_info_topic_name"                      default="/camera/rgb/camera_info"/>>
  <arg name="img_topic_out"                               default="/qr_detector/front"/>>
  <arg name="camera_type"                                 default="front"/>>
  <arg name="QR_code_side_size"                           default="0.3"/>>
  <arg name="detect_state"                                default="true"/>>

  <node pkg="object_detector" name="qr_detector_front" type="qr_detector.py" output="screen">
        <param name="camera_image_topic"                                                value="$(arg camera_image_topic)"/>
  <param name="camera_info_topic_name"                                      value="$(arg camera_info_topic_name)"/>
  <param name="img_topic_out"                                                 value="$(arg img_topic_out)"/>
  <param name="camera_type"                                                     value="$(arg camera_type)"/>
  <param name="QR_code_side_size"                                                 value="$(arg QR_code_side_size)"/>
  <param name="detect_state"                                                  value="$(arg detect_state)"/>
  </node>

</launch>
